kmp-library/src/jvmMain/kotlin/com/novoda/aws/v4/signer/hash/Hmac256Encoder.kt

```diff
@@ -0,0 +1,17 @@
+package com.novoda.aws.v4.signer.hash
+
+import javax.crypto.Mac
+import javax.crypto.spec.SecretKeySpec
+
+private const val HMAC_SHA256 = "HmacSHA256"
+
+actual object Hmac256Encoder {
```

Comment:
The `JVM` Hmac256Encoder implementation using `javax.crypto.Mac` as it was done previously in the `java-library`.

Reply:
:+1: